--- zshrc_original	2016-04-29 23:46:00.033213467 +0300
+++ zshrc_new	2016-04-29 23:44:44.323213466 +0300
@@ -1,23 +1,21 @@
-export ZSH=$HOME/.config/oh-my-zsh
+export ZSH=/home/cli3mo/.config/oh-my-zsh
 
-export PATH="/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:$HOME/.local/bin/:$HOME/.local/share/npm/bin"
+export PATH="/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/cli3mo/.local/bin/:/home/cli3mo/.local/share/npm/bin"
 env | grep -o 'TMUX=' &> /dev/null && export TERM='screen-256color' || export TERM='xterm-256color'
 export EDITOR="nvim"
 export ARCHFLAGS="-arch x86_64"
-export SSH_KEY_PATH="$HOME/.ssh/id_rsa"
+export SSH_KEY_PATH="/home/cli3mo/.ssh/id_rsa"
 
 export XDG_CONFIG_HOME=$HOME/.config
 export XDG_CACHE_HOME=$HOME/.cache
 export XDG_DATA_HOME=$HOME/.local/share
 
-export XAUTHORITY=$XDG_CONFIG_HOME/xorg/xauthority
-export SCRIPTS_FOLDER=$HOME/src/scripts
-export GTK2_RC_FILES=$XDG_CONFIG_HOME/gtk-2.0/gtkrc
+export SCRIPTS_FOLDER=/home/cli3mo/src/scripts
 export TMUX_TMPDIR=$XDG_DATA_HOME/tmux
-export npm_config_userconfig=$XDG_CONFIG_HOME/npm/npmrc
+export npm_config_userconfig=/home/cli3mo/.config/npm/npmrc
 export LESSHISTFILE=/dev/null
 
-plugins=(sudo archlinux extract)
+plugins=(archlinux extract)
 
 ZSH_THEME="powerlevel9k/powerlevel9k"
 
@@ -40,5 +38,3 @@
 source $XDG_CONFIG_HOME/zsh/powerlevel.zsh
 source $ZSH/oh-my-zsh.sh
 source /usr/share/zsh/plugins/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
-
-[[ -z $DISPLAY && $XDG_VTNR -eq 1 ]] && startx $HOME/.config/xorg/xinitrc; :
